spotishell/Public/Add-MultipleFollowedArtists.ps1

```diff
@@ -0,0 +1,39 @@
+function Add-MultipleFollowedArtists {
```

Comment:
I'd like to avoid functions that do the same thing as another except they take multiple values. The Add-FollowedArtist should be able to accept an array of artists or a single artist.